app/routes/__index/dns-records/index.tsx

```diff
@@ -55,21 +55,20 @@ export const action = async ({ request }: ActionArgs) => {
     });
   }
 
-  if (dnsRecord.status !== 'active') {
-    throw new Response('DNS Record is not active, action forbidden', {
-      status: 409,
-    });
-  }
-
   switch (intent) {
     case 'renew-dns-record':
+      if (dnsRecord.status !== 'active') {
+        throw new Response('DNS Record is not active, action forbidden', {
+          status: 409,
```

Comment:
Should this error code be `403` (forbidden) or `409` (conflict)? A user could mistakenly press the renew button in the front end, thinking it would refresh the status.